docs/key_features.md

```diff
@@ -0,0 +1,28 @@
+# Key Features of `scores`
+
+## Data Handling
+
+- Works with labelled, n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the dimensionality, data size and data structures commonly used for:
+  - gridded Earth system data (e.g., numerical weather prediction models)
+  - tabular, point, latitude/longitude or site-based data (e.g., forecasts for specific locations).
+- Handles missing data, masking of data and weighting of results.
+- Supports [xarray](https://xarray.dev/) datatypes, and works with [NetCDF4](https://doi.org/10.5065/D6H70CW6), [HDF5](https://github.com/HDFGroup/hdf5), [Zarr](https://zarr.dev) and [GRIB](https://codes.wmo.int/grib2) data sources among others. 
+
+## Usability
+
+- A companion Jupyter Notebook [tutorial](project:./tutorials/Tutorial_Gallery.md) for each metric and statistical test that demonstrates its use in practice.
+- [Over 50 metrics, statistical techniques and data processing tools](included.md), including:
+  - commonly-used metrics (e.g., mean absolute error (MAE), root mean squared error (RMSE))
+  - novel scores not commonly found elsewhere (e.g., FIxed Risk Multicategorical (FIRM) score ([Taggart et al., 2022](https://doi.org/10.1002/qj.4266)), Flip-Flop Index ([Griffiths et al., 2019](https://doi.org/10.1002/met.1732), [2021](https://doi.org/10.1071/ES21010)))
+  - complex scores (e.g., threshold-weighted continuous ranked probability score (twCRPS))
+  - statistical tests (e.g., the Diebold-Mariano ([Diebold & Mariano, 1995](https://doi.org/10.3386/t0169)) test, with both the Harvey et al. ([1997](https://doi.org/10.1016/S0169-2070(96)00719-4)) and Hering & Genton ([2011](https://doi.org/10.1198/tech.2011.10136)) modifications).
```

Comment:
(1) Should there also be a sub-dot-point about tools for pre-processing data?

(2), If yes, is there a preference for one of the following:
- Tools to pre-process data
- Tools for pre-processing data
- Tools for pre-processing data, including data matching, discretisation and cumulative density function manipulation
- Tools for pre-processing data in a variety of formats, including cumulative distribution functions (CDF).
- Something else?

Reply:
Note to self: if add tools to pre-process data sub-dot-point, make sure to remove full stop at end of proceeding dot point.

Reply:
If this sub-dot-point is included, I think I am leaning towards one of the following:
- tools for pre-processing data; including data matching, discretisation and cumulative density function manipulation.
- tools for pre-processing data - including data matching, discretisation and cumulative density function manipulation.

But I could be swayed in another direction, and am also open to alternative suggestions.

Reply:
Have raised this question in a new issue, so will resolve the conversation here.